README.md

```diff
@@ -435,18 +450,22 @@ Some resources possess an emoticon to help you understand which type of content
 **[⬆ back to top](#table-of-contents)**
 
 ---
+
 ## SEO
 
 * [ ] **Google Analytics:** ![High](http://res.cloudinary.com/djnyaloac/image/upload/v1508238836/level-checklist-high.png) Google Analytics is installed and correctly configured.
 * [ ] **Headings logic:** ![Medium](http://res.cloudinary.com/djnyaloac/image/upload/v1508238836/level-checklist-medium.png) Heading text helps to understand the content in the current page.
 * [ ] **sitemap.xml:** ![High](http://res.cloudinary.com/djnyaloac/image/upload/v1508238836/level-checklist-high.png) A sitemap.xml exists and was submit in Google Search Console (ex:Google Webmaster Tools)
 * [ ] **robots.txt:** ![High](http://res.cloudinary.com/djnyaloac/image/upload/v1508238836/level-checklist-high.png) The robots.txt is not blocking webpages
+
 Link your website to the Google webmaster tools
```

Comment:
This was probably meant to be a link to Google Webmaster tools. If so, let me know so that I update it.